ports/directxsdk/portfile.cmake

```
@@ -3,9 +3,9 @@ vcpkg_fail_port_install(ON_TARGET "LINUX" "OSX" "UWP" "ANDROID" ON_ARCH "arm")
 message(WARNING "Build ${PORT} is deprecated, untested in CI, and requires the use of the DirectSetup legacy REDIST solution. See https://aka.ms/dxsdk for more information.")
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://archive.org/download/dxsdk_2010/DXSDK_Jun10.exe"
+    URLS "https://download.microsoft.com/download/A/E/7/AE743F1F-632B-4809-87A9-AA1BB3458E31/DXSDK_Jun10.exe"
     FILENAME "DXSDK_Jun10.exe"

```

We need to rename the downloaded file to avoid colliding with existing cached copies
```suggestion
    FILENAME "DXSDK_Jun10_24e1e9bda.exe"
```